lib/iex/lib/iex/config.ex

```diff
@@ -136,8 +137,21 @@ defmodule IEx.Config do
   end
 
   def width() do
+    columns = columns()
+    case Application.get_env(:iex, :width) do
+      nil -> default_width()
```

Comment:
I think there is no need for `default_width/0` function as you already have `columns` value from the call on line#140.